client/src/components/Common/MiddleTruncatedText/index.jsx

```diff
@@ -4,6 +4,9 @@ import { getEndString } from '../../../helpers/format';
 import './MiddleTruncatedText.scss';
 
 export const MiddleTruncatedText = ({ children, end, placement = 'top' }) => {
+	if (!children) {
+		return null;
+	}
```

Comment:
This indicates something loosely validated from parent, also an not a best practice for minor component.
I think from the parent's component we should check for proper condition before rendering `<MiddleTruncatedText />`.

Reply:
Just remove this